src/ui/common/src/components/AnnouncementBanner/AnnouncementBanner.tsx

```diff
@@ -153,8 +153,8 @@ export const AnnouncementBanner: React.FC<AnnouncementBannerProps> = ({
           icon={faXmark}
           onClick={() => {
             if (onClose) {
-              onClose();
```

Comment:
Going to put this back, this change isn't needed, was debugging something else here.